### DISCUSSION TITLE
Is it possible to use mpv without DE?

### DISCUSSION THREAD

**Discussion by demosspro** [Joshua-Riek/ubuntu-rockchip#384] [General]
Hello dear Joshua!
First - i wanna say you a BIG thanks for your work. It is pretty good images to start work with that SBC.
Can i ask you - is it possible to obtain MPV with normal worked futures on that board with image without DE?
Thank for answere

**Comment by Joshua-Riek** [comment]
I have no idea, my only experience with MPV is to playback videos in a DE environment. I don't know much about the inner workings of MPV or video decoding/encoding.

**Comment by demosspro** [comment]
Joshua thanks for the answer! I will look for those who may know the answer to this question. But maybe they will be here too and can tell us the answer. Their advice may be to turn off DE and use a few specific options to play the video file by MPV using a command from the terminal.

**Comment by Ferenc-** [comment]
@demosspro In theory you would have to build `sdl2` with `--enable-video-directfb`, and then build `mpv` with `--enable-sdl2`

**Reply by demosspro** [reply]
@Ferenc- Thank you for response. I will try it and them write here result.
